core/language.go

```diff
@@ -5,6 +5,15 @@ import (
 	sitter "github.com/smacker/go-tree-sitter"
 )
 
+// ImportContents represents the contents of an import node
+// It represents the ready to use content of import nodes which may
+// exhibit different forms in different languages
+type ImportContents struct {
```

Comment:
I think we should avoid introducing this as a core language concern. It will introduce ambiguity wrt. `ImportNode`. If this is required, we can move this to `helpers`.